src/components/DebatecoreFooter.tsx

```diff
@@ -0,0 +1,71 @@
+import { useLang } from "@/lib/useLang";
+import { Lexend } from "next/font/google";
+
+const lexend = Lexend({ subsets: ["latin"] });
+
+type footerlink = {
+  text: string;
+  href: string;
+};
+
+const DebatecoreFooter = (props: {}) => {
+  const boldfont = lexend.className;
+
+  const footerlinks: footerlink[] = [
+    // {
+    //   text: useLang("footerAbout"),
+    //   href: "/about",
+    // },
+    // {
+    //   text: useLang("footerMission"),
+    //   href: "https://debateco.re/about",
+    // },
+    // {
+    //   text: useLang("footerSendFeedback"),
+    //   href: "https://debateco.re/sendfeedback?from=toolsfooter",
+    // },
+    {
+      text: `${useLang("footerLicense")} (AGPLv3)`,
+      href: "https://github.com/debatecore/debate-tools/blob/master/LICENSE",
+    },
+    {
+      text: useLang("footerSourceCode"),
+      href: "https://github.com/debatecore/debate-tools",
+    },
+  ];
+
+  return (
+    <>
+      <div className="flex flex-col md:flex-row justify-center items-center gap-4 p-2 mx-auto">
+        <div>
+          <a
+            href="https://debateco.re"
+            className={`${boldfont} tracking-wide text-transparent bg-clip-text bg-gradient-to-r from-violet-600 to-pink-400 from-30% to-70%`}
+          >
+            debatecore
+          </a>
+        </div>
+        <div className="w-1 h-1 hidden md:block bg-neutral-500 rounded" />
```

Comment:
Is this dot meant to separate every link in the footer, or only the logo from the rest? I don't insist on either, just making sure this is intentional.

Reply:
logo from the list, and it disappears on mobile since it works better that way